src/main/java/com/kodilla/ecommercee/entity/Item.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.ecommercee.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+public class Item {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "cart_id", nullable = false)
+    private Cart cart;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "product_id", nullable = false)
+    private Product product;
```

Comment:
Tak samo tutaj, jeśli ktoś dodał Item to Product już powinien dla niego istnieć, a nie być zapisywany